content-hour-row.php

```diff
@@ -66,136 +66,133 @@
 											<b>Today</b><br/>
 										<?php
 											$curDay = $now;
-											
-											$temp = $post;									
-											
-											//echo "S: ".date("Y-m-d", $today)." ";
-											
-											$message = getMessageDay($locationId, $today);
-											
-											if ($message != "" ) {
-												
-												if ($showMobileSpecial == 1) {
+
+											$temp = $post;
+
+											// echo "S: ".date("Y-m-d", $today)." "; .
```

Comment:
I don't like the period at the end of this - it will mess up the function if uncommented.